pkg/sdk/tasks_gen_test.go

```diff
@@ -552,3 +554,66 @@ func TestTasks_Execute(t *testing.T) {
 		assertOptsValidAndSQLEquals(t, opts, "EXECUTE TASK %s RETRY LAST", id.FullyQualifiedName())
 	})
 }
+
+func TestParseTaskSchedule(t *testing.T) {
```

Comment:
In the comment for this function `America/Los_Angeles` is mentioned and it is not here in any test

Reply:
True, added